rest-api/tools/remove_service_account_keys.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# remove keys for service accounts that start with 'awardee-' that are older than 3 days.
```

Comment:
I'm not sure we need to keep this script, since we're building an endpoint to do the work. If we do want to keep it, you may want to document here that there's an endpoint which does the same thing, and add something to the tools README about it.

Reply:
done

Reply:
done